exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -96,6 +107,23 @@
 
   private Server embeddedJetty;
 
+  private static File tmpJavaScriptDir;
+
+  public File getTmpJavaScriptDir() {
+    if (tmpJavaScriptDir == null) {
```

Comment:
How many times this would be called? I am just wondering why we have this null check.

Reply:
I wanted to bundle the creation as a part of the dir creation, so this check is for that.